<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W_AddTeamSouvenirsClass</class>
 <widget class="QWidget" name="W_AddTeamSouvenirsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>W_AddTeamSouvenirs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="L_TeamName">
       <property name="font">
        <font>
         <family>Cascadia Mono SemiLight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;TeamName&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_Total">
       <property name="font">
        <font>
         <family>Cascadia Mono SemiLight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Total: $0.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="SA_Souvenirs"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
